include/dt-bindings/pinctrl/nrf-pinctrl.h

```diff
@@ -51,6 +51,15 @@
  * @{
  */
 
+/** UART TX */
```

Comment:
That is, add a comment here saying "If you add functions here, make sure to keep the binding documentation for the psel property in sync"

Reply:
I think we should just let Doxygen render dt-bindings public headers to avoid duplicate information/manual handling.

Reply:
How are users going to find that information from the bindings index, then?

Reply:
`Refer to `include/dt-bindings/pinctrl/nrf-pinctrl.h for available functions`

Reply:
OK, I suppose.